README.md

```diff
@@ -232,6 +234,7 @@ You can specify the browser and Node.js versions by queries (case insensitive):
 You can add `not ` to any query.
 
 [`caniuse-lite/data/regions`]: https://github.com/ben-eb/caniuse-lite/tree/master/data/regions
+[`caniuse-lite/data/features`]: https://github.com/ben-eb/caniuse-lite/tree/master/data/features
```

Comment:
It is hard to use this dir because you can’t read the description in those files.

It will be better to show that it is a `feat` parameter on Can I Use page.

Reply:
Fixed, I've added an explanation of the `feat` parameter, and left the link as reference.